### PR TITLE
Feat responsive website

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Media breakpoints are added to make website more responsive
2. Document layout is refactored to best suit the new changes

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -1,42 +1,38 @@
...
 .badge-text {
-    font-size: 1em;
-    padding: 0.1rem 0.35rem;
-    border-radius: 0.05em;
-    position: absolute; 
-    top: 0%;
-    right: 0%;
-    transform: translate(50%, -50%); 
+  font-size: 1em;
+  padding: 0.1rem 0.35rem;
```
Comment (fahd23) [inline components/badge/badge.css:31]: Is there any reason to give padding in rems here, we can give it in px right if it is less than 10px

File: components/components.css [hunk 2]
```diff
@@ -13,52 +13,84 @@
...
-    html {
-        font-size: 50%;
-    }
-}
\ No newline at end of file
+  html {
+    font-size: 50%;
+  }
+} */
+
```
Comment (fahd23) [inline components/components.css]: If it is not required, remove this commented code

File: main.css [hunk 3]
```diff
@@ -2,69 +2,70 @@
...
 
+.border-box {
+  border: 2px solid black;
+  border-radius: 5px;
+}
+
+/* .responsive-images-container {
+  aspect-ratio: 2/0.5;
+
+} */
```
Comment (fahd23) [inline main.css:71]: same here

**Review by fahd23 — APPROVED** [review]
Given some suggestions, looks good

File: components/badge/badge.css [hunk 4]
```diff
@@ -1,42 +1,38 @@
...
+  border-radius: 0.05em;
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  transform: translate(50%, -50%);
+  background-color: var(--info__bg-color);
 }
```
Comment (mariyasada) [inline components/badge/badge.css:38]: don't mix conventions like hyphen and underscore. use either hyphen or underscore

File: components/components.css [hunk 5]
```diff
@@ -13,52 +13,84 @@
...
+    font-size: 52.5%;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  html {
+    font-size: 57.5%;
+  }
+}
+
```
Comment (mariyasada) [inline components/components.css]: liked that you have changed the font size according the screens

**Review by mariyasada — APPROVED** [review]
this code looks fine to me

**Pull request merged**